### ISSUE TITLE
Add missing test coverage

### ISSUE THREAD

**Issue by bnonni** [TBD54566975/incubation-dcx#115]

**Comment by MuskanSaxena6** [comment]
Am i supposed to set these values according to me ISSUER_WEB5_PASSWORD=' ', ISSUER_WEB5_RECOVERY_PHRASE=' ' ?

**Comment by bnonni** [comment]
@MuskanSaxena6 yes, you can set them manually in a `.env` or if you look at https://github.com/TBD54566975/incubation-dcx/blob/main/packages/issuer/src/config.ts you'll see the config object is looking for `process.env.ISSUER_WEB5_PASSWORD` and
`process.env.ISSUER_WEB5_RECOVERY_PHRASE` in the env or if `null`, coalesce to a function from the `Mnemonic` class to fill it.

Setting these vars in `.env` is more for BYO-Keys. If you don't have a pre-existing Agent DID recovery phrase and password, I recommend allowing the `Mnemonic` class auto-populating them or you can explicitly define them using that same class.

**Comment by MuskanSaxena6** [comment]
> > Am i supposed to set these values according to me: ISSUER_WEB5_PASSWORD=' ', ISSUER_WEB5_RECOVERY_PHRASE=' ' ?
> 
> @MuskanSaxena6 you can set them manually in a `.env` or if you look at [issuer/src/config.ts](https://github.com/TBD54566975/incubation-dcx/blob/main/packages/issuer/src/config.ts) you'll see the config object is looking for `process.env.ISSUER_WEB5_PASSWORD` and
> `process.env.ISSUER_WEB5_RECOVERY_PHRASE` in the env or if `null`, coalesce to a function from the `Mnemonic` class to fill it. Setting these vars in `.env` is more for BYO-Keys. If you don't have a pre-existing Agent DID recovery phrase and password, I recommend allowing the `Mnemonic` class to auto-populate them. You can also use that same class to explicitly define those vars wherever you need them.
> 
> **Recommendation**
> Because this is a test coverage issue, I recommend either letting the `config.ts` code handle the auto-population or importing `Mnemonic` and explicitly defining a new  `ISSUER_WEB5_PASSWORD` and `ISSUER_WEB5_RECOVERY_PHRASE` in the test case you're writing. If you look in the [issuer/tests](https://github.com/TBD54566975/incubation-dcx/blob/main/packages/issuer/tests) folder and specifically the [issuer/tests/issuer.spec.ts](https://github.com/TBD54566975/incubation-dcx/blob/main/packages/issuer/tests/issuer.spec.ts) test file, you'll find good starting points for understanding what has been setup already and where to go from here. Looking at `issuer.spec.ts` and `config.spec.ts`, you can see there's only 2 tests for the `issuer` package. Ideally, we'd have coverage for all the parts of the `issuer` package code + the `applicant`, `common` and `server` code bases.
> 
> Feel free to send me your questions in the TBD Discord channel for [#dcx](https://discord.gg/EfkzHxqT). I tend to not see GitHub comments as quickly.

@bnonni thanks a lot , you could have easily given me the answer in a single line but you explained it in such a detailed manner, I'm learning a lot!!

**Comment by MuskanSaxena6** [comment]
@bnonni I've tried to increase the test coverage but , I've failed to do the same,you can unassign me , so that others can help with this issue.Thank you got being so patient I learned a lot from working on this issue.
